### PR TITLE
tickets/DM-29739

### PR DESCRIPTION
**Pull request by mareuter**
Fix Jenkinsfile.conda.

### CODE REVIEW

File: Jenkinsfile.conda [hunk 1]
```diff
@@ -1,2 +1,2 @@
 @Library('JenkinsShared')_
-CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig")
+CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig", arch="noarch")
```
Comment (couger01) [inline Jenkinsfile.conda]: This syntax doesn't work the way that you expect it to. Use this instead. 
```suggestion
CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig", "noarch")
```

**Review by couger01 — CHANGES_REQUESTED** [review]
Needs one change otherwise looks good.

File: Jenkinsfile.conda (hunk 1 above)
Comment (mareuter) [reply Jenkinsfile.conda]: OK. Fixed that.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**